src/salame/Salame.cpp

```diff
@@ -27,20 +27,31 @@ SalameModule (Hipace* hipace, const int n_iter, const bool do_advance, int& last
     }
     last_islice = islice;
 
+    last_islice = islice;
+
```

Comment:
Isn't this a duplication of the line above?
```suggestion
```

Reply:
I blame git